agent/host.go

```diff
@@ -28,6 +28,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/dselans/dmidecode"
```

Comment:
you need to "govendor fetch github.com/dselans/dmidecode" to get this package added to vendor/vendor.json

Reply:
ok

Reply:
solved